ios/brave-ios/Sources/Brave/Frontend/Browser/Translate/TranslateToast.swift

```diff
@@ -0,0 +1,346 @@
+// Copyright 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import BraveStrings
+import BraveUI
+import DesignSystem
+import SwiftUI
+
+private struct TranslationOptionsView: View {
+  @Environment(\.dismiss)
+  private var dismiss
+
+  @State
+  private var searchText = ""
+
+  @Binding
+  var language: Locale.Language?
+
+  var body: some View {
+    NavigationStack {
+      List {
+        ForEach(filteredLanguages, id: \.self) { language in
+          Button(
+            action: {
+              self.language = language
+              dismiss()
+            },
+            label: {
+              Text(languageName(for: language))
+            }
+          )
+        }
+      }
+      .listStyle(.plain)
+      .navigationTitle(Strings.BraveTranslate.languageSelectionButtonTitle)
+      .navigationBarTitleDisplayMode(.inline)
+      .toolbar {
+        ToolbarItem(placement: .navigationBarLeading) {
+          Button(Strings.CancelString) {
+            dismiss()
+          }
+        }
+      }
+    }
+    .searchable(
+      text: $searchText,
+      placement: .navigationBarDrawer(displayMode: .always),
+      prompt: Strings.BraveTranslate.searchInputTitle
+    )
+  }
+
+  private var filteredLanguages: [Locale.Language] {
+    return searchText.isEmpty
+      ? languages
+      : languages.filter({
+        languageName(for: $0).lowercased().contains(searchText.lowercased())
+      })
```

Comment:
We should start using https://developer.apple.com/documentation/foundation/nsstring/1409742-localizedstandardcompare to compare user input

Reply:
Sorry wrong link, should be https://developer.apple.com/documentation/foundation/nsstring/1416328-localizedstandardcontains